Jenkinsfile

```
@@ -1,4 +1,15 @@
 #!/usr/bin/env groovy
 
-/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+import java.util.Collections
+
+// Valid Jenkins versions for test
+def testJenkinsVersions = [ '2.164.3', '2.176.4', '2.190.3', '2.204.6', '2.222.4', '2.235.1', '2.243' ]
+Collections.shuffle(testJenkinsVersions)
+
+// Test plugin compatibility to subset of Jenkins versions
+subsetConfiguration = [ [ jdk: '8',  platform: 'windows', jenkins: testJenkinsVersions[0], javaLevel: '8' ],
+                        [ jdk: '8',  platform: 'linux',   jenkins: testJenkinsVersions[1], javaLevel: '8' ],
+                        [ jdk: '11', platform: 'linux',   jenkins: testJenkinsVersions[2], javaLevel: '8' ]
+                      ]
+
+buildPlugin(configurations: subsetConfiguration, failFast: false)
```

When I submitted this pull request, I was not aware that the technique used in this Jenkinsfile is incompatible with Jenkins incrementals.  This change is not needed to remove the bundling of JGit from the plugin.  If this plugin were to consider implementing incrementals, this would need to return to the earlier implementation.
```suggestion
/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
buildPlugin()
```